lms/static/sass/base/_grid-settings.scss

```diff
@@ -7,7 +7,7 @@ $border-box-sizing: false;
 
 
 /* Breakpoints */
-$mobile: new-breakpoint(max-width 320px 4);
-$tablet: new-breakpoint(min-width 321px max-width 768px, 8);
+$mobile: new-breakpoint(max-width 540px 4);
```

Comment:
I've made these changes so that breakpoints in here correspond to the breakpoints used on the edx.org site.
The breakpoints used on the edx.org site are:
``` scss
// breakpoints for course cards
$bp-xl-min: 1180px;
$bp-l-min: 980px;
$bp-m-min: 768px;
$bp-s-min: 320px;
$bp-s-max: 320px;
$bp-m-max: 768px;
$bp-one-col-min: 540px;
```